### ISSUE TITLE
Template creation after install - R3.1 RC1

### ISSUE THREAD

**Issue by canihavesomecoffee** [QubesOS/qubes-issues#1524]
Hello,

I've installed the RC1 on my laptop today. Initial installation went all fine, but when I booted for the first time, it gave some errors while trying to create the templates. It referred me to the /var/log/salt/minion log, which contains the following entries:

```
2015-12-18 01:25:12,054 [salt.loaded.int.module.cmdmod][ERROR   ] Command 'systemctl status salt-minion.service' failed with return code: 3
2015-12-18 01:25:12,056 [salt.loaded.int.module.cmdmod][ERROR   ] output: salt-minion.service - The Salt Minion
   Loaded: loaded (/usr/lib/systemd/system/salt-minion.service; disabled)
   Active: inactive (dead)
2015-12-18 01:25:12,090 [salt.loaded.int.module.cmdmod][ERROR   ] Command 'systemctl is-active salt-minion.service' failed with return code: 3
2015-12-18 01:25:12,091 [salt.loaded.int.module.cmdmod][ERROR   ] output: unknown
2015-12-18 01:25:32,527 [salt.state       ][CRITICAL] Rendering SLS 'dom0:qvm.sys-firewall' failed: Problem running salt function in Jinja template: User 'user' is not available
/var/cache/salt/minion/files/dom0/qvm/template.jinja(105):
---
[...]
  {%- set force = defaults.get('force', vm.get('force', False)) %}

  {#- Only attempt to install the VM if it is missing (not installed) to prevent
   # changing an existing configuration, unless ``force`` is True.
   #}
  {%- if force or salt['qvm.check'](vm.name, 'missing').passed() -%}    <======================
    {{- state_debug(defaults, vm) }}
    {{- state_vm(vm) }}
  {%- else -%}
    {{- skip(vm) }}
  {%- endif -%}
[...]
---
(this last critical one is repeated 5 more times)
```

If more information is required, please let me know.

**Comment by Truthlighting** [comment]
I had a similar or the same problem after install:

My problem was that the sys-net and sys-firewall templates never got installed. During the final phase after install, it errored out stating: 

```
Executing qubes configuration failure! 
Qubes initial configuration failed. 
Login the the system and check /var/log/salt/minion for details. 
You can retry configuration by calling 'sudo qubesctl state.highstate' in dom0 (you will get detailed state there).
```

So then I ran `sudo qubesctl state.highstate` in `dom0` and, in short, received this error (and a number of other errors; one for each of the vms that were not installed):

```
[CRITICAL] Rendering SLS 'dom-:qvm.sys-firewall' failed: Problem running salt function in Jinja template: user 'user' is not available
```

I then ran: `adduser user` and `sudo usermod -a -G qubes user`. I was then able to successfully run:
 `sudo qubesctl state.highstate` which corrected a number of issues with the VMs.

Unfortunately, now there's another issue. There's missing KDE menu items associated with the initially not-installed system VMs e.g. "Template: sys-firewall"/Template: sys-net" and associated application menu items.
